docs/guides/operators/grpc.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "@grpc"
+sidebar_position: 4
+---
+
+The **@grpc** operator is an essential GraphQL custom directive designed to interface with gRPC services. It allows GraphQL queries to directly invoke gRPC services, thereby bridging two powerful technologies. This directive is particularly useful when integrating GraphQL with microservices that expose gRPC endpoints.
+
+Here's an overview of each argument within the **@grpc** directive:
+
+## Overview of parameters
+
+### `service`
+
+Specifies the gRPC service to be called. This should match the service name as defined in the `.proto` file.
+
+### `method`
+
+Indicates the specific gRPC method to be invoked within the specified service.
+
+### `body`
+
+Outlines the arguments for the gRPC call, which can be static or dynamic.
+
+### `baseURL`
+
+Defines the base URL for the gRPC API. If omitted, the default URL defined in the `@upstream` operator is used.
+
```

Comment:
I suggest not to have sections with overview and detailed explanation separately and put them inside one heading section. Because in current approach it could be unclear for user that more detailed information is somewhere else and since the heading name are the same we can't create shareable link to detailed heading (that link will always point to first occurrence)